niworkflows/engine/splicer.py

```diff
@@ -0,0 +1,123 @@
+"""Workflow splicing operations."""
+
+import typing as ty
+
+from nipype.pipeline import Workflow
```

Comment:
would this not conflict with the `LiterateWorkflow` defined in workflow? I think we want to change the import in `.workflow` so it imports from here instead of form nipype.